_benchmark/commands/command-flags.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: default
+title: Command flags
+nav_order: 51
+parent: Command reference
+---
+
```

Comment:
To keep consistent with other pages, we can add a title here:
`### Command line flags`

Reply:
Good catch!